mm2src/coins/qrc20/history.rs

```diff
@@ -299,13 +292,13 @@ impl Qrc20Coin {
             let internal_id = TxInternalId::new(tx_hash.clone(), receipt.output_index, log_index as u64);
 
             let from = if is_sender_contract(&script_pubkey) {
```

Comment:
I find the naming a bit confusing - in E/QRC20 and ETH context, the sender/receiver definitions are associated with sender/receiver naming in the etomic swap smart contract. Maybe the `transferred_from_contract` and `transferred_to_contract` will be more clear?

Reply:
Thanks, now it looks clearer